src/ccontrol/UserQueryDrop.cc

```diff
@@ -115,18 +115,18 @@ void UserQueryDrop::submit() {
         // Has it disappeared already?
         LOGS(_log, LOG_LVL_ERROR, "database disappeared from CSS");
         std::string message = "Unknown database " + _dbName;
-        _messageStore->addMessage(-1, 1051, message, MessageSeverity::MSG_ERROR);
+        _messageStore->addMessage(-1, "CSS", 1051, message, MessageSeverity::MSG_ERROR);
```

Comment:
I know you didn't add those, though, I'm curious, what's the meaning of these hardwired "codes" such as `1051`? I'm seeing those in the new `QMessages` table as well. What are they telling us, and what are we supposed to do about these numbers?

Reply:
My guess is that it's a mysql error code. mysql error 1051 is unknown table, which seems appropriate.

Reply:
Using this code here is incorrect for two reasons. First of all, the previous line is complaining about a database:
```
std::string message = "Unknown database " + _dbName;
```
 Meanwhile, the MySQL error code 1051 corresponds to the macro definition for `ER_BAD_TABLE_ERROR`, which is posted when the table is not found. See: https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html#error_er_bad_table_error

The second comment is our code should not use hard coded MySQL error codes. If one needs to report MySQL-specific code then the different code `ER_BAD_DB_ERROR` should be used:
```
// Third party headers
#include <mysql/mysqld_error.h>
#include <mysql/errmsg.h>

std::string message = "Unknown database " + _dbName;
_messageStore->addMessage(-1, "CSS", ER_BAD_DB_ERROR, message, MessageSeverity::MSG_ERROR);
```